### PR TITLE
736d.tt append to README.md

### PR DESCRIPTION
**Pull request by kanglingchu**
append **736d**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # graphql.js
 
-> GitHub GraphQL API client for browsers and Node (**at branch `736d`**)
+> GitHub GraphQL API client for browsers and Node (**at branch `736d`;`736d.tt`**)
```
Comment (kangshumin) [inline README.md]: tt, please tell me **your name**?

Comment (kanglingchu) [reply README.md]: why u intend to known my name?

Comment (kangshumin) [reply README.md]: ok,ok, u ox

**Comment by kangshumin** [comment]
i would merge your pull request! :)

**Comment by kanglingchu** [comment]
fine!

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # graphql.js
```
Comment (kangshumin) [inline README.md:1]: tt, it's here?

Comment (kanglingchu) [reply README.md:1]: yeah!

**Comment by kangshumin** [comment]
tt, u append yourself *full name*

**Pull request merged**